ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -3214,6 +3219,7 @@ static int zend_jit_check_exception_undef_result(dasm_State **Dst, const zend_op
 
 static int zend_jit_trace_begin(dasm_State **Dst, uint32_t trace_num, zend_jit_trace_info *parent, uint32_t exit_num)
 {
+	|	ENDBR
```

Comment:
This is also wrong. ``ENDBR`` for "root" traces is generated in ``zend_jit_prologue()``. This ``ENDBR`` for "side" traces should be generated only if ``(parent != NULL)``

Reply:
That makes sense. I will add a judgment.